### ISSUE TITLE
controlsd: resume from stop doesn't work with stock long

### ISSUE THREAD

**Issue by sshane** [commaai/openpilot#33850]

**Comment by BBBmau** [comment]
I'm experiencing this also except it's on experimental. This is from master on commit af73d6084d5ac77935282442c909432eebd676d1 i believe.

**Comment by sshane** [comment]
Can you post a route where this happens?

**Comment by BBBmau** [comment]
@sshane should be present here, if not I can get a more recent route

9a31d18cd1a79c69/00000001--79aa41f7e0/53

https://connect.comma.ai/9a31d18cd1a79c69/00000001--79aa41f7e0/3225/3237

**Comment by sshane** [comment]
The model in experimental mode failing to resume is different than controlsd setting the carControl.cruiseControl.resume signal BTW, that's what this issue is for

**Comment by BBBmau** [comment]
> The model in experimental mode failing to resume at traffic lights is different than controlsd setting the carControl.cruiseControl.resume signal based on the planner/lead BTW, that's what this issue is for

yeah the route is wrong, I'll be providing the route with the actual issue later today. Sorry for the confusion.

**Comment by BBBmau** [comment]
@sshane looking to see what needs to be changed. A bit interesting how on resume it goes to the saved speed but is quickly reverted to be 65

![image](https://github.com/user-attachments/assets/0e761422-f082-4d57-aae8-2f3fc1110f80)

**Comment by sshane** [comment]
This works on a recent route, not sure if this was a real issue. Closing
